lib/models/polls/poll_option_model.dart

```diff
@@ -0,0 +1,62 @@
+import 'dart:convert';
+
+class PollOptionModel {
+  final int id;
+  final String title;
+  final int votes;
+
+  PollOptionModel({
+    required this.id,
+    required this.title,
+    required this.votes,
+  });
+
+  PollOptionModel copyWith({
+    int? id,
+    String? title,
+    int? votes,
+  }) {
+    return PollOptionModel(
+      id: id ?? this.id,
+      title: title ?? this.title,
+      votes: votes ?? this.votes,
+    );
+  }
+
+  Map<String, dynamic> toMap() {
+    return {
+      'id': id,
+      'title': title,
+      'votes': votes,
+    };
+  }
+
+  factory PollOptionModel.fromMap(Map<String, dynamic> map) {
+    return PollOptionModel(
+      id: map['id']?.toInt() ?? 0,
+      title: map['title'] ?? '',
+      votes: map['votes']?.toInt() ?? 0,
+    );
+  }
+
+  String toJson() => json.encode(toMap());
+
```

Comment:
Are we not sticking to the json_serialisation package?

Reply:
As discussed, it will be better than the extensions used.

Reply:
Yes true, we are. This is just for the purpose of the poll screen. Will change when working on the data layer.